docs/reference/ml/apis/put-dfanalytics.asciidoc

```diff
@@ -43,7 +43,18 @@ and {stack-ov}/built-in-roles.html[Built-in roles].
   
 `dest` (Required)::
   (object) The destination configuration, consisting of `index` and optionally 
-  `results_field` (`ml` by default).
+  `results_field` (`ml` by default). The destination index will be automatically 
+  created if it does not exist. The `index.number_of_shards` and 
+  `index.number_of_replicas` settings of the source index will be copied over the 
+  destination index. When the source index matches multiple indices, these 
+  settings will be set to the maximum values found in the source indices.
+  
+  The mappings of the source indicies are also attempted to being copied over
```

Comment:
rephrase: `also attempted to being copied over` -> `also attempted to be copied over` ?